modules/aggregation-layers/src/cpu-grid-layer/cpu-grid-layer.js

```diff
@@ -76,7 +76,7 @@ export default class CPUGridLayer extends AggregationLayer {
     };
     const attributeManager = this.getAttributeManager();
     attributeManager.add({
-      positions: {size: 3, accessor: 'getPosition' /* , type: GL.DOUBLE, fp64: false*/}
+      positions: {size: 3, accessor: 'getPosition'}
```

Comment:
- `type: GL:DOUBLE`?
- Why does this layer need the `attributeManager` for positions? It will create a WebGL buffer that it never uses.

Reply:
as discussed offline, using attribute manager for just positions and reading it from attributes array. This allows any AttributeManager features/optimization for position attribute available for cpu aggregation, we can remove this later if no such items are available. (Also original intetion was data filtering, but that has been made exclusive for GPU aggregation only.)

Reply:
`GL.DOUBLE` packs a Float64Array instead of Float32Array. You lose precision without it in LNGLAT mode.